packages/@ourworldindata/grapher/src/tooltip/TooltipContents.tsx

```diff
@@ -169,18 +170,23 @@ export class TooltipTable extends React.Component<TooltipTableProps> {
                             name,
                             focused,
                             blurred,
+                            striped,
                             annotation,
                             values,
                             notice,
                             swatch = "transparent",
                         } = row
                         const [_m, seriesName, seriesParenthetical] =
-                            name.match(/^(.*?)(\(.*)?$/) ?? []
+                            name.match(/^(.*?)(\(([^()]*)\))?$/) ?? []
```

Comment:
nit: you got an extra inner unused capture group
```suggestion
                            name.match(/^(.*?)(\([^()]*\))?$/) ?? []
```